src/collections/_documentation/platforms/python/bottle.md

```diff
@@ -0,0 +1,63 @@
+---
+title: Bottle
+sidebar_order: 5
+---
+
+{% version_added 0.7.9 %}
+
+
+<!-- WIZARD -->
+*Import name: `sentry_sdk.integrations.bottle.BottleIntegration`*
+
+The Bottle integration adds support for the [Bottle Web Framework](https://bottlepy.org/).
```

Comment:
Could you specify here which versions of bottle the integration supports? We should have a sentence like that for some of the existing integrations.

Reply:
A note was added. I also discovered that it doens't work well with the development version of bottle (0.13). Probably I need to redo some of the patching.